### ISSUE TITLE
Consolidated Tracker for Example Discussions

### ISSUE THREAD

**Issue by alizma** [METHODS-Group/DRDMannTurb#49]
Now that the examples actually work and embed properly into the documentation, they should come with discussions. Specifically, the models used, data pre-processing, and results from fits and wind field generation should all be included. The following is a list of the example notebooks that will be used in the documentation. Please refer to the [sphinx-gallery documentation](https://sphinx-gallery.github.io/dev/syntax.html) for how these should be written to ensure they render properly. Note that math comments follow the usual Sphinx conventions. 

NOTE: this issue will not be closed until approval that the documentation properly describes the package's functionality is given.

- [x] Pure Mann model fit
- [x] Synthetic Data Fit (example 1 from original paper) 
- [x] Changing architecture with synthetic fit 
- [x] Changing loss function with synthetic fit 
- [x] Interpolating and filtering custom data and then fitting 
- [x] Custom data fit without pre-processing (example 2 from paper)
- [x] Linear-regression based Mann model fit 
- [x] Wind generation from a pre-trained model (based on Gaussian random fields)
